ansible/roles/debops.dovecot/defaults/main.yml

```diff
@@ -297,12 +385,21 @@ dovecot_imap_listeners: [ 'imap', 'imaps' ]
 # for more details.
 dovecot_imap_config_map: {}
 
+                                                                   # ]]]
+# .. envvar:: dovecot_imap_combined_config_map [[[
+#
+# Combined IMAP configuration properties.
```

Comment:
Is it necessary to add this to the defaults? Are you using own custom variables to combine?

Reply:
Yes it helps.

This is part of my inventory config for dovecot:

```
dovecot_imap_config_map:

  login-service:
    service_count: 1
    process_min_avail: 1

  service:
    process_limit: 512

  protocol:
    mail_max_userip_connections: 15
```